WolvenKit.App/Controllers/RED4Controller.cs

```diff
@@ -194,33 +194,40 @@ public async Task<bool> InstallProjectHot()
 
         var hotDirectory = Path.Combine(_settingsManager.GetRED4GameRootDir(), "archive", "pc", "hot");
 
-        // create hot directory
-        if (!Directory.Exists(hotDirectory))
+        try
```

Comment:
why the try clause?

Reply:
Because I ran into an exception while working on the other thing, which completely broke the project bar buttons until I reloaded the project